Sources/xcproj/XCScheme.swift

```diff
@@ -398,6 +460,10 @@ final public class XCScheme {
                 .all?
                 .map(TestableReference.init) ?? []
             self.macroExpansion = try BuildableReference(element: element["MacroExpansion"]["BuildableReference"])
+
+            if let commandlineOptions = element.children.first(where: { $0.name == "CommandLineArguments" }) {
```

Comment:
If @yonaskolb is right regarding the XML API to find the children don't forget to update this one as well.